spec/project_spec.rb

```diff
@@ -676,6 +676,22 @@ class InvalidErrorHandler; end
       expect(spy_logger).to have_received(:log).once.with(Logger::INFO, "Feature 'boolean_single_variable_feature' is enabled for user 'test_user'.")
     end
 
+    it 'should return true if the user is bucketed into a feature rollout even if featureEnabled property is false' do
```

Comment:
This is not the case. We should always be checking the `featureEnabled` property. The reason is because in the future we might conceive of a world where there "could" be multiple variations. In the datafile the property will ALWAYS be set to true if there is traffic allocated to it.